demisto_sdk/commands/content_graph/interface/graph.py

```diff
@@ -0,0 +1,63 @@
+
+from abc import ABC, abstractmethod
+from typing import Any, Dict, List, Optional
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+
+from demisto_sdk.commands.content_graph.common import ContentTypes, Rel
+
+class ContentGraphInterface(ABC):
+    @abstractmethod
+    def create_indexes_and_constraints(self) -> None:
+        pass
+
+    @abstractmethod
+    def create_nodes(self, nodes: Dict[ContentTypes, List[Dict[str, Any]]]) -> None:
+        pass
+
+    @abstractmethod
+    def create_relationships(self, relationships: Dict[Rel, List[Dict[str, Any]]]) -> None:
+        pass
+
+    @abstractmethod
+    def validate_graph(self) -> None:
+        pass
+    
+    @abstractmethod
+    def get_packs_content_items(self, marketplace: MarketplaceVersions):
+        pass
+
+    @abstractmethod
+    def get_all_integrations_with_commands(self):
+        pass
+
+    @abstractmethod
+    def delete_all_graph_nodes_and_relationships(self):
```

Comment:
```suggestion
    def clear_graph(self):
```
Or something similar. No need to list (in the method name) the types that you'll delete.